src/features/offer/helpers/renderOfferPageTestUtil.tsx

```diff
@@ -66,10 +43,16 @@ export async function renderOfferBodyPage(
   { isLoggedIn } = { isLoggedIn: true }
 ) {
   mockedOffer = { ...offerResponseSnap, ...extraOffer }
-  mockedUser = { ...baseUser, ...user }
+  const mockedUser = { ...beneficiaryUser, ...user }
 
   const setIsLoggedIn = jest.fn()
-  mockUseAuthContext.mockImplementation(() => ({ isLoggedIn, setIsLoggedIn, user: mockedUser }))
+  mockUseAuthContext.mockImplementation(() => ({
+    isLoggedIn,
+    setIsLoggedIn,
+    user: mockedUser,
+    isUserLoading: false,
+    refetchUser: jest.fn(),
+  }))
```

Comment:
FYI : mauvaise pratique car `mockImplementation` sera appelée dans les it()…

Mais tant pis, c'était déjà présent et la PR est déjà conséquente 😟